PyInstaller/compat.py

```diff
@@ -360,7 +360,13 @@ def exec_command(*cmdargs, **kwargs):
     """
 
     encoding = kwargs.pop('encoding', None)
-    out = subprocess.Popen(cmdargs, stdout=subprocess.PIPE, **kwargs).communicate()[0]
+    try:
+        out = subprocess.Popen(cmdargs, stdout=subprocess.PIPE, **kwargs).communicate()[0]
+    except OSError as e:
+        print('--' * 20, file=sys.stderr)
+        print("Error running '%s': (%s) " % (" ".join(cmdargs), str(e)), file=sys.stderr)
```

Comment:
I would change this
- into two lines (break after the colon)
- not put the error-message into parents
- leave off the `str()` call, since print converts to string anyway,

So this would become
```
print("Error running '%s':" % " ".join(cmdargs), file=sys.stderr)
print(e, file=sys.stderr)
```